package.json

```diff
@@ -126,6 +127,7 @@
     "gulp-clean": "^0.4.0",
     "gulp-util": "^3.0.8",
     "highcharts": "^6.0.7",
+    "highlight.js": "^9.17.1",
```

Comment:
`highlight.js` is a dependency in case that we are using it in barista. Please reinstall it as dependency